extensions/interactions/SetInput/directives/oppia-interactive-set-input.component.ts

```diff
@@ -79,6 +79,9 @@ export class InteractiveSetInputComponent implements OnInit {
   }
 
   updateAnswer(answer: SetInputAnswer): void {
+    const hasDuplicates = this.hasDuplicates(answer);
+    this.errorMessage = hasDuplicates ?
+    'Oops, it looks like your answer has duplicates!' : '';
```

Comment:
Follow existing indentation patterns.

```suggestion
    const hasDuplicates = this.hasDuplicates(answer);
    this.errorMessage = (
      hasDuplicates ? 'Oops, it looks like your answer has duplicates!' : '');
```

Reply:
> Follow existing indentation patterns.

Understood, thanks!